### PR TITLE
feat: add Rocket.chat support

### PR DESCRIPTION
**Pull request by dmitry-kovalev**

### CODE REVIEW

File: pkg/services/rocketchat/rocketchat.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	service.Logger.SetLogger(logger)
+	service.config = &Config{}
+	if err := service.config.SetURL(configURL); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Send a notification message to Slack
```
Comment (piksel) [inline pkg/services/rocketchat/rocketchat.go:36]:
```suggestion
// Send a notification message to Rocket Chat
```

File: pkg/services/rocketchat/rocketchat_config.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	return &url.URL{
+		Host:       config.Host,
+		Path:       fmt.Sprintf("hooks/%s/%s", config.TokenA, config.TokenB),
+		Scheme:     Scheme,
+		ForceQuery: false,
+	}
+}
+
+// https://chat.gravitel.ru/hooks/8eGdRzc9r4YYNyvge/2XYQcX9NBwJBKfQnphpebPcnXZcPEi32Nt4NKJfrnbhsbRfX
+// rocketchat://rocketchat.my-domain.com/tokenA/tokenB
```
Comment (piksel) [inline pkg/services/rocketchat/rocketchat_config.go:33]: These comments should probably not be here?

File: pkg/services/rocketchat/rocketchat_test.go [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+func TestRocketchat(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Shoutrrr Rocketchat Suite")
+}
+
+var _ = Describe("the rocketchat service", func() {
+	BeforeSuite(func() {
+		service = &Service{}
+		envRocketchatURL, _ = url.Parse(os.Getenv("SHOUTRRR_MATTERMOST_URL"))
```
Comment (piksel) [inline pkg/services/rocketchat/rocketchat_test.go:26]:
```suggestion
		envRocketchatURL, _ = url.Parse(os.Getenv("SHOUTRRR_ROCKETCHAT_URL"))
```

**Review by piksel — CHANGES_REQUESTED** [review]
Some copy/paste errors and needs some description. 

Adding docs on how to set it up would be nice too, but is not a requirement. Linking to the official documentation and/or describing the service would suffice as a description.

**Comment by dmitry-kovalev** [comment]
I've resolved all comments in one commit and added documentation on Rocket.chat integration.

File: docs/services/overview.md [hunk 4]
```diff
@@ -17,3 +17,4 @@ Click on the service for a more thorough explanation.
 | [Hangouts Chat](./hangouts.md)    | *hangouts://chat.googleapis.com/v1/spaces/FOO/messages?key=bar&token=baz*                                                                       |
 | [Zulip Chat](./zulip.md)          | *zulip://__`bot-mail`__:__`bot-key`__@__`zulip-domain`__/?stream=__`name-or-id`__&topic=__`name`__*                                             |
 | [Join](./not-documented.md)       | *join://shoutrrr:__`api-key`__@join/?devices=__`device1`__[,__`device2`__, ...][&icon=__`icon`__][&title=__`title`__]*                          |
+| [Rocketchat](./not-documented.md) | *rocketchat://[__`username`__@]__`rocketchat-host`__/__`token`__[/__`channel`&#124;`@recipient`__]*                                                               |
```
Comment (piksel) [inline docs/services/overview.md:20]:
```suggestion
| [Rocketchat](./rocketchat.md) | *rocketchat://[__`username`__@]__`rocketchat-host`__/__`token`__[/__`channel`&#124;`@recipient`__]*                                                               |
```

**Review by piksel — APPROVED** [review]
Looks good!

The documentation should probably be linked instead of `not-documented.md` though.

**Pull request merged**